### PR TITLE
update posts docs

### PR DESCRIPTION
**Pull request by kazzarin**
This should complete #8

### CODE REVIEW

File: blueprint/posts/comment-likes.apib [hunk 1]
```diff
@@ -1,4 +1,132 @@
...
++ Response 404 (application/vnd.api+json)
+    + Attributes (Error404)
+
++ Response 409 (application/vnd.api+json)
+    + Attributes (Error409)
+
++ Response 500 (application/vnd.api+json)
+    + Attributes (Error500)
+
+### Update Resource [PATCH]
```
Comment (wopian) [inline blueprint/posts/comment-likes.apib]: As there's nothing that can be modified - it either exists (liked) or doesn't, it's better to omit this request type completely

File: blueprint/posts/comments.apib [hunk 2]
```diff
@@ -1,7 +1,21 @@
 ## Comments [/comments/{id}]
 
+**Authorisation**
+
+Authenticated | Role      | GET | POST | PATCH | DELETE
+------------: | --------: | :-: | :--: | :---: | :----:
+No            | None      | Y   | N    | N     | N
+Yes           | None      | Y   | Y    | N*    | Y
```
Comment (wopian) [inline blueprint/posts/comments.apib]: This should be `Y*` not `N*` as you *can* update the resource, just for a limited period.

File: blueprint/posts/post-follows.apib [hunk 3]
```diff
@@ -12,12 +24,12 @@
...
+            + count: 2261 (number, required)
         + links (object)
             + first: `https://kitsu.io/api/edge/post-follows?page%5Blimit%5D=10&page%5Boffset%5D=0`
             + prev: `https://kitsu.io/api/edge/post-follows?page%5Blimit%5D=10&page%5Boffset%5D=0` - Omitted if on first page
             + next: `https://kitsu.io/api/edge/post-follows?page%5Blimit%5D=10&page%5Boffset%5D=10` - Omitted if on last page
-            + last: `https://kitsu.io/api/edge/post-follows?page%5Blimit%5D=10&page%5Boffset%5D=2066`
+            + last: `https://kitsu.io/api/edge/post-follows?page%5Blimit%5D=10&page%5Boffset%5D=2251`
 
 + Response 400 (application/vnd.api+json)
     + Attributes (Error400)
```
Comment (wopian) [inline blueprint/posts/post-follows.apib:35]: Can you remove the patch request below ❤️

File: blueprint/posts/post-likes.apib [hunk 4]
```diff
@@ -12,12 +24,12 @@
...
+            + count: 1774249 (number, required)
         + links (object)
             + first: `https://kitsu.io/api/edge/post-likes?page%5Blimit%5D=10&page%5Boffset%5D=0`
             + prev: `https://kitsu.io/api/edge/post-likes?page%5Blimit%5D=10&page%5Boffset%5D=0` - Omitted if on first page
             + next: `https://kitsu.io/api/edge/post-likes?page%5Blimit%5D=10&page%5Boffset%5D=10` - Omitted if on last page
-            + last: `https://kitsu.io/api/edge/post-likes?page%5Blimit%5D=10&page%5Boffset%5D=1600072`
+            + last: `https://kitsu.io/api/edge/post-likes?page%5Blimit%5D=10&page%5Boffset%5D=1774239`
 
 + Response 400 (application/vnd.api+json)
     + Attributes (Error400)
```
Comment (wopian) [inline blueprint/posts/post-likes.apib:35]: Remove patch request below

File: blueprint/posts/posts.apib [hunk 5]
```diff
@@ -1,5 +1,17 @@
 ## Posts [/posts/{id}]
 
+**Authorisation**
+
+Authenticated | Role      | GET | POST | PATCH | DELETE
+------------: | --------: | :-: | :--: | :---: | :----:
+No            | None      | Y   | N    | N     | N
+Yes           | None      | Y   | Y    | N*    | Y
```
Comment (wopian) [inline blueprint/posts/posts.apib]: `Y*` instead of `N*`

**Comment by kazzarin** [comment]
Fixed :ok_hand:

**Pull request merged**